pageserver/src/tenant/secondary/downloader.rs

```diff
@@ -1146,12 +1146,14 @@ impl<'a> TenantDownloader<'a> {
         layer: HeatMapLayer,
         ctx: &RequestContext,
     ) -> Result<Option<HeatMapLayer>, UpdateError> {
-        // Failpoint for simulating slow remote storage
+        // Failpoints for simulating slow remote storage
         failpoint_support::sleep_millis_async!(
             "secondary-layer-download-sleep",
             &self.secondary_state.cancel
         );
 
+        pausable_failpoint!("secondary-layer-download-pausable");
```

Comment:
Couldn't the existing failpoint above be used?  (the sleep_millis failpoints are usable for pausing if you pass a large value in)

Reply:
Perhaps. I was unsure if I can turn it off while it's sleeping. I don't want the test to wait on the failpoint.